controllers/sriovoperatorconfig_controller.go

```diff
@@ -59,6 +59,7 @@ type SriovOperatorConfigReconciler struct {
 	Scheme         *runtime.Scheme
 	PlatformHelper platforms.Interface
 	FeatureGate    featuregate.FeatureGate
+	KubeClient     client.Client
```

Comment:
Can we be more specific and use `mgr.GetAPIReader()`. It would be clearer if anyone else needs to do operations avoiding the cache,

```suggestion
	uncachedAPIReader     client.Reader
```

Reply:
OK, I will modify this part and resubmit the PR again